samples/net/updatehub/prj.conf

```diff
@@ -1,5 +1,6 @@
 # Turn on the updatehub
 CONFIG_UPDATEHUB=y
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Why is this change needed?

Reply:
This is need as the `select NEWLIB_LIBC` in `lib/updatehub/Kconfig` has been changed to `depends on !MINIMAL_LIBC`.

The reason behind the Kconfig change is that `select on` is not allowed on config symbols in a choice.